src/card.css

```
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+ 
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-block {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  padding-bottom: 40px;
```

```suggestion
  margin-bottom: 40px;
```
distance between two blocks should be set using `margin`